### PR TITLE
Overloop - New Components

### LINKED ISSUE #3712: Overloop
### App
Overloop

### Triggers
* Contact Created
* Contact Replied
* Deal Created
* Deal Lost
* Deal Won
* New Exclusion List Item
* Campaign Created
* List Created
* Pipeline Created
* Stage Created
* User Created
* Workflow Created
* Organization Created

### Actions
* Add Contact to Campaign
* Create Contact
* Create Deal
* Create Organization
* Enroll Contact
* Enroll Deal
* Enroll Organization
* Exclude Contact
* Mark Deal as Lost
* Mark Deal as Won
* Move Deal
* Search Email
* Update Contact
* Update Deal
* Update Organization
* Search Contact
* Search Deal
* Search Organization

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #3712

### CODE REVIEW

File: components/overloop/actions/mark-deal-won/mark-deal-won.mjs
Comment (alysonturing) [inline components/overloop/actions/mark-deal-won/mark-deal-won.mjs:19]: is that correct? this action is `mark-deal-won` but the method called is `markDealLost`

File: components/overloop/overloop.app.mjs
Comment (alysonturing) [inline components/overloop/overloop.app.mjs:308]: I believe that this method is not being used

**Review by alysonturing — CHANGES_REQUESTED** [review]
Thanks for your contribution, please check the requested changes.

**Comment by GTFalcao** [comment]
Hi @michelle0927 , I see the requested change has been addressed, so this is good to go!

@alysonturing since you had previously requested changes, Github will require your approval before the PR can be merged.

**Review by GTFalcao — APPROVED** [review]

**Comment by alysonturing** [comment]
Sorry, it's done!

**Pull request merged**
